tests/integration/security/rbac/v2/basic/testdata/istio-rbac-grpc-rules.yaml.tmpl

```diff
@@ -0,0 +1,50 @@
+# istio-rbac-grpc-rules.yaml to enforce access control for gRPC services using Istio RBAC v2 rules.
+
+# For service a:
+# * Allow b to call a's Echo method.
+# * Do not allow c to talk to a (since NonExistingMethod does not exist).
+# * Allow d to call any methods of a.
+
+apiVersion: "rbac.istio.io/v1alpha1"
+kind: ServiceRole
+metadata:
+  name: access-a-grpc-echo
+  namespace: {{ .Namespace }}
+spec:
+  rules:
+    - paths: ["/proto.EchoTestService/Echo"]
+---
+apiVersion: "rbac.istio.io/v1alpha1"
+kind: ServiceRole
+metadata:
+  name: not-access-a-grpc
+  namespace: {{ .Namespace }}
+spec:
+  rules:
+    - paths: ["/proto.EchoTestService/NonExistingMethod"]
+---
+apiVersion: "rbac.istio.io/v1alpha1"
+kind: ServiceRole
+metadata:
+  name: access-a-grpc-any
+  namespace: {{ .Namespace }}
+spec:
+  rules:
+    - paths: ["*"]
+---
+apiVersion: "rbac.istio.io/v1alpha1"
+kind: AuthorizationPolicy
```

Comment:
1. I will add grpc test for v1alpha1 in a separate PR.
2. That should not matter, right? Since it select all services (that includes `a`). But I can specify `a` if you think it's better